docker/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine
+FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine as builder
 
-COPY ./compiler /RESTler/compiler
-COPY ./restler /RESTler/restler
+RUN apk add --no-cache python3 py3-pip
+RUN ln -s /usr/bin/python3 /usr/bin/python
 
-#Get the source code for the engine, and
-#copy source code to the image
-#RESTler Python dependency
-RUN apk add --no-cache python3
-RUN python3 -m ensurepip
-RUN pip3 install --upgrade pip
-RUN pip3 install requests
-RUN pip3 install requests
-RUN pip3 install applicationinsights
-COPY ./engine /RESTler/engine
-RUN python3 -m compileall -b /RESTler/engine
-COPY ./resultsAnalyzer /RESTler/resultsAnalyzer
+COPY src ./src
+COPY restler ./restler
+COPY build-restler.py .
+
+RUN python3 build-restler.py --dest_dir /build
+
```

Comment:
The container build should fail if the build command fails - in my testing, it did not.

Reply:
that is because of an odd method of catching build failures. I've added a simple replacement which uses the builder exit code to determine failure

Reply:
Looks good, thanks!